### PR TITLE
Add Terraform deployment instructions!

### PR DESCRIPTION
**Pull request by Clete2**
Quick question -- do you have documentation of the SAR template items somewhere? I wanted to update those too.

### CODE REVIEW

**Comment by alexcasalboni** [comment]
Thanks 🎉 

>  do you have documentation of the SAR template items somewhere?

Do you mean the stack parameters?

**Comment by Clete2** [comment]
@alexcasalboni Yep, the params. I may be blind.

**Comment by alexcasalboni** [comment]
No worries :)

Here you go: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/README-INPUT-OUTPUT.md#state-machine-input-at-deployment-time

File: README-DEPLOY.md [hunk 1]
```diff
@@ -128,6 +128,29 @@ You can find the open-source project and the instructions to deploy it here: [ma
...
+Simply add the `aws_serverlessapplicationrepository_cloudformation_stack` resource below to your Terraform code and deploy as usual through `terraform apply`.
+
+```hcl
+resource "aws_serverlessapplicationrepository_cloudformation_stack" "lambda-power-tuning" {
+  name           = "lambda-power-tuner"
+  application_id = "arn:aws:serverlessrepo:us-east-1:451282441545:applications/aws-lambda-power-tuning"
+  capabilities   = ["CAPABILITY_IAM"]
+
+  parameters = {
+    # Many of these parameters are optional and are only shown here for demonstration purposes
```
Comment (alexcasalboni) [inline README-DEPLOY.md:142]: They're all optional :) Maybe we could comment the whole parameters section?

I'm doing the same here: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/scripts/deploy-sar-app.yml

Comment (Clete2) [reply README-DEPLOY.md]: Good idea. I commented them out and updated to include a link to that page with all the params that you provided.

File: README-DEPLOY.md [hunk 2]
```diff
@@ -128,6 +128,30 @@ You can find the open-source project and the instructions to deploy it here: [ma
...
 ![Power Tuner UI](https://github.com/mattymoomoo/aws-power-tuner-ui/blob/master/imgs/website.png?raw=true)
 
+## Option 6: deploy the SAR app with Terraform
+
+Simply add the `aws_serverlessapplicationrepository_cloudformation_stack` resource below to your Terraform code and deploy as usual through `terraform apply`.
+
+```hcl
+resource "aws_serverlessapplicationrepository_cloudformation_stack" "lambda-power-tuning" {
+  name           = "lambda-power-tuner"
+  application_id = "arn:aws:serverlessrepo:us-east-1:451282441545:applications/aws-lambda-power-tuning"
```
Comment (alexcasalboni) [inline README-DEPLOY.md:138]: Does this deploy the latest version by default? Can you define the `SemanticVersion` explicitly as well?

Comment (Clete2) [reply README-DEPLOY.md]: Yes it does. I updated to add a commented-out version as an example.

**Comment by Clete2** [comment]
I just added a link to the Terraform docs for the SAR CF stack resource too @alexcasalboni

**Comment by alexcasalboni** [comment]
Looks great! I'm ready to merge, if that's ok for you.

**Comment by Clete2** [comment]
Good to go

**Pull request merged**